src/cdk/tree/tree-using-legacy-key-manager.spec.ts

```diff
@@ -16,6 +16,7 @@ describe('CdkTree when provided LegacyTreeKeyManager', () => {
 
     fixture = TestBed.createComponent(SimpleCdkTreeApp);
     fixture.detectChanges();
+    fixture.detectChanges();
```

Comment:
These consecutive `detectChanges` calls seem a bit suspicious. Do we know why they're necessary?

Reply:
the ViewChildren queries weren't being set for some reason without this; AOT + signals seems to work

Reply:
setting AOT broke a lot of tests, though I think I've solved a few of the change detection issues now